assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -137,11 +137,10 @@ const fetchCreatePropertyStore = createFetchStore( {
 
 // Actions
 const WAIT_FOR_PROPERTIES = 'WAIT_FOR_PROPERTIES';
-const START_MATCHING_PROPERTY = 'START_MATCHING_PROPERTY';
-const STOP_MATCHING_PROPERTY = 'STOP_MATCHING_PROPERTY';
+const FINISH_MATCHING_PROPERTY = 'FINISH_MATCHING_PROPERTY';
 
 const baseInitialState = {
-	isMatchingProperty: false,
+	finishedMatchingProperty: undefined,
```

Comment:
Shouldn't this be `false` right on load? I don't think this makes sense to be `undefined` initially, since it's definitely not finished on load. The `undefined` here would prevent the strict `=== false` check above from being truthy.